vaygrant/Vagrantfile

```diff
@@ -14,7 +14,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
   # Create a forwarded port mapping which allows access to a specific port
   # within the machine from a port on the host machine.
-  config.vm.network :forwarded_port, guest: 9000, host: 9000 # For accessing octoparts
+#  config.vm.network :forwarded_port, guest: 9000, host: 9000 # For accessing octoparts
```

Comment:
No it was getting in the way on my machine. When I had Vagrant running (for the DB and Memcached, etc), my Play on my host machine wasn't able to bind to 9000 :(

I should revert this.